test.java

```diff
@@ -6,16 +6,50 @@
 	
 	
 	@Test

```

Comment:
Please read about [parameterized tests](https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests) and rewrite this.